app/src/main/res/layout/item_feed.xml

```diff
@@ -15,20 +15,21 @@
         android:layout_height="wrap_content"
         android:background="?attr/selectableItemBackground"
         >
-        <TextView
+        <android.support.text.emoji.widget.EmojiTextView
```

Comment:
Thanks!  👍 
Please use TextView for this 🙇

Reply:
OK!
I will try it. :smile: 
Because the date does not use emoji, right? :worried:

Reply:
Yes 👍 That's right 😄